moved docs to rst file (#5)
christophfroehlich/ros2_control_demos: doc/index.rst

  cd ros2_control_demos
  docker build . -t ros2_control_demos -f Dockerfile/Dockerfile

To View the Robot

> ************

Reviewer: ```suggestion
To view the robot
^^^^^^^^^^^^^^^^^
```
We have to use the same markup as other headers of the same hierarchy, and it has to be as long as the heading itself,

```
/workspaces/ros2_rolling_ws/src/control.ros.org/doc/ros2_control_demos/doc/index.rst:161: WARNING: Title underline too short.
To View the Robot
************
/workspaces/ros2_rolling_ws/src/control.ros.org/doc/ros2_control_demos/doc/index.rst:160: CRITICAL: Title level inconsistent:
To View the Robot
************
```

Docker now allows us to run the demo without the GUI if configured properly. Now we can view the robot by the following procedure:

> After having `ros2 rolling installed on your system<https://docs.ros.org/en/rolling/Installation.html>`, we can use the joint state publisher gui package to give manual joint values to the robot. To install the package you can run:

Reviewer: ```suggestion
After having `ROS 2 installed <https://docs.ros.org/en/rolling/Installation.html>`__ on your local system (not inside the docker),  we can use ``rviz2`` to visualize the robot state and ``joint_state_publisher_gui`` package to give manual joint values to the robot. To install the package you can run:
```
Usage of the correct version of ROS 2 https://shouldiputaspacebetweenrosand2.com/ :smile: 

markup for link is with trailing `__` and a space before the `<`

.. code-block:: shell
  

>   sudo apt-get install -y ros-rolling-joint-state-publisher-gui

Reviewer: ```suggestion
  sudo apt-get install -y ros-${ROS_DISTRO}-joint-state-publisher-gui ros-${ROS_DISTRO}-rviz2
```

Then we are ready to bring up all the components to view the robot. Let's start with the docker container by running the following command:

.. code-block:: shell

  docker run -it --rm --name ros2_control_demos --net host ros2_control_demos ros2 launch ros2_control_demos_example_1 view_robot.launch.py gui:=false

> Now, we need to start rviz2 to view the robot as well as joint state publisher gui, each in their own terminals after sourcing our ros2 installation.

Reviewer: ```suggestion
Now, we need to start ``rviz2`` to view the robot as well as ``joint_state_publisher_gui``, each in their own terminals after sourcing our ROS 2  installation.
```

Terminal 1:

> .. code-block:: shell

Reviewer: ```suggestion
Terminal 1:

.. code-block:: shell
```
there has to be a blank line before the code block

>   source /opt/ros/rolling/setup.bash

Reviewer: ```suggestion
  source /opt/ros/${ROS_DISTRO}/setup.bash
```
otherwise backports are a nightmare

  ros2 run joint_state_publisher_gui joint_state_publisher_gui

Terminal 2:

> .. code-block:: shell

Reviewer: ```suggestion
Terminal 2:

.. code-block:: shell
```

>   source /opt/ros/rolling/setup.bash

Reviewer: ```suggestion
  source /opt/ros/${ROS_DISTRO}/setup.bash
```

  cd ~/ros2_ws
  rviz2 -d src/ros2_control_demos/example_1/description/rviz/rrbot.rviz

> Now, you can see the robot moving by changing the values of the joints by moving the sliders around in the joint state publisher gui.

Reviewer: ```suggestion
Now, you can see the robot moving by changing the values of the joints by moving the sliders around in the ``joint_state_publisher_gui``.
```

> To Run the ros2 control demo
> ************

Reviewer: ```suggestion
To run the ros2_control demos
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
same as above
